level-2/index.html

```diff
@@ -130,9 +130,10 @@ <h4>
             <dfn>canShare()</dfn> method
           </h4>
           <p>
-            <code>canShare(</code><var>data</var><code>)</code> MUST return
-            <code>true</code> unless
-            <code>share(</code><var>data</var><code>)</code> would reject with
+            <code>canShare(</code><var>data</var><code>,</code>
```

Comment:
What are your thoughts on making `canShare` not take `options` (since, at least for now, there are no options that should affect the shareability of the data).

I could go either way on this.

Reply:
That works.

We could make `canShare(null, {message: ""})` be a way to detect if `message` is supported, but I don't think it is worthwhile.